CHANGES.md

```diff
@@ -1,9 +1,9 @@
 ### Apache MetaModel _WIP_
 
- * [METAMODEL-1151] - Created DataContextFactory implementations for each remaining module.
```

Comment:
We shouldn't remove this item

Reply:
Oh, sure. That was my merging error. Updated.